api/brands/views.py

```diff
@@ -0,0 +1,73 @@
+
+from django.core.exceptions import ObjectDoesNotExist
+from rest_framework import generics
+from rest_framework import permissions as drf_permissions
+from rest_framework.exceptions import NotFound
+
+from framework.auth.oauth_scopes import CoreScopes
+from osf.models import Brand
+from api.base import permissions as base_permissions
+from api.base.pagination import MaxSizePagination
+from api.base.views import JSONAPIBaseView
+from api.brands.serializers import BrandSerializer
+
+
+class BrandMixin(object):
+    """Mixin with convenience method get_brand
+    """
+
+    brand_lookup_url_kwarg = 'brand_id'
+
+    def get_brand(self):
+        brand_id = self.kwargs[self.brand_lookup_url_kwarg]
+
+        try:
+            inst = Brand.objects.get(id=int(brand_id))
+        except ObjectDoesNotExist:
+            raise NotFound
+
+        return inst
+
+
+class BrandList(JSONAPIBaseView, generics.ListAPIView, BrandMixin):
+    """The documentation for this endpoint can be found [here](https://developer.osf.io/#operation/institutions_list).
+    """
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+    )
+
+    required_read_scopes = [CoreScopes.ALWAYS_PUBLIC]
+    required_write_scopes = [CoreScopes.NULL]
+    model_class = Brand
+
+    pagination_class = MaxSizePagination
```

Comment:
I don't think we need this to be a MaxSizePagination. Let's go with the standard json api pagination for this. As a general rule, MaxSizePagination is only for use in situations where: a) we aren't doing a db query to get the list of items; or b) I'm forced to allow it by someone who's in a position to force it. Generally, though, pagination is good stuff.